sdk/nodejs/tests/automation/localWorkspace.spec.ts

```diff
@@ -1462,7 +1462,9 @@ describe("LocalWorkspace", () => {
     });
     it(`correctly sets config on multiple stacks concurrently`, async () => {
         const dones = [];
-        const stacks = ["dev", "dev2", "dev3", "dev4", "dev5"];
+        const stacks = ["dev", "dev2", "dev3", "dev4", "dev5"].map(x =>
```

Comment:
This one, however, gave me an error to say that the organization _had to be_ `organization`. Not sure what's different here, but this passes locally now.